### DISCUSSION TITLE
Crossover Support

### DISCUSSION THREAD

**Discussion by itisnajim** [alvr-org/ALVR#2500] [Q&A] [answered]
I know that SteamVR doesn’t show up on [Crossover](https://www.codeweavers.com/crossover), and ALVR could be a great alternative for macOS users. However, when trying to run ALVR Streamer on Crossover, nothing happens. If someone on the dev team with access to a Mac could help investigate, it would be greatly appreciated!

**Comment by zmerp** [comment]
ALVR could not work on Crossover because it's is written as a DLL loaded by SteamVR

**Reply by itisnajim** [reply]
what stops SteamVR from working on Crossover, if u have any idea ?

**Reply by zmerp** [reply, marked as answer]
I believe SteamVR relies on specific GPU vendor drivers and configurations. For example regarding frame timing and reprojection. On linux these features can be disabled but the resulting support is very poor
